docs/src/main/sphinx/connector/elasticsearch.rst

```diff
@@ -374,6 +374,8 @@ Additionally, the following options need to be configured appropriately:
 ================================================ ==================================================================
 Property name                                    Description
 ================================================ ==================================================================
+``elasticsearch.aws.credentials-provider``       Fully-qualified class name of a custom credentials provider.
```

Comment:
I don't think we want to support this in the ES connector (and possibly elsewhere). 

How would a user provide the implementation for such a class? What interface is that class supposed to implement? What are the backward compatibility guarantees as the connector evolves?

Reply:
@ksobolew What if we would make a new library for AWS credential provider to be customizable. So in certain use case we can decide what features are used. In case of elastic we would not expose possibility of configuring credentials provider. Same with region in other place.

That way extracting common library won't affect the UX.